crates/metadata/src/utils.rs

```diff
@@ -56,3 +57,11 @@ where
 
     deserializer.deserialize_str(Visitor)
 }
+
+pub fn trim_extra_whitespace(item: &'static str) -> &'static str {
```

Comment:
```suggestion
pub fn trim_extra_whitespace(item: &str) -> &str {
```
Lifetimes can be elided